recipes/libcugraph/build.sh

```diff
@@ -27,10 +27,10 @@ cd ${LIBCUGRAPH_BUILD_DIR}
 cmake -DCMAKE_INSTALL_PREFIX=${INSTALL_PREFIX} \
       ${GPU_ARCH} \
       -DDISABLE_DEPRECATION_WARNING=${BUILD_DISABLE_DEPRECATION_WARNING} \
+      -DCUDA_TOOLKIT_ROOT_DIR=${CUDA_HOME} \
       -DCMAKE_BUILD_TYPE=${BUILD_TYPE} \
       -DBUILD_STATIC_FAISS=${BUILD_STATIC_FAISS} \
       -DBUILD_TESTS=${BUILD_CPP_TESTS} \
       -DBUILD_CUGRAPH_MG_TESTS=${BUILD_CPP_MG_TESTS} \
-      "${CUGRAPH_SRC_DIR}/cpp" \
-      || (cat "${CUGRAPH_SRC_DIR}/cpp/build/CMakeFiles/CMakeOutput.log" && cat "${CUGRAPH_SRC_DIR}/cpp/build/CMakeFiles/CMakeError.log" && exit 1)
```

Comment:
Would it be alright if we kept this? It makes it easier to see what CMake is running into issues with

Reply:
I'll add it back but I'm going to experiment with adding formatting to make it obvious it's a log dump and not cmake stdout,  since I think it was causing confusion (see my comment above).